securedrop/translations/ar/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,1256 @@
+
+#: source_app/main.py:140
+msgid "Thanks! We received your document."
+msgstr "شكرا! لقد استلمنا مستندك."
+
+#: source_app/main.py:142
+msgid "Thanks! We received your message and document."
+msgstr "شكرا! لقد استلمنا مستندك ورسالتك."
+
+#: source_app/main.py:178
+msgid "Reply deleted"
+msgstr "تم حذف الرد"
+
+#: source_app/main.py:195
+msgid "All replies have been deleted"
+msgstr "تم حذف جميع الردود"
+
+#: source_app/main.py:209
+msgid "Sorry, that is not a recognized codename."
+msgstr "نأسف, هذا الاسم الرمزي غير معترف به."
+
+#: source_templates/base.html:6 source_templates/index.html:4
+msgid "Protecting Journalists and Sources"
+msgstr "حماية الصحفيين و المصادر"
+
+#: source_templates/base.html:24 source_templates/base.html:43
+#: source_templates/index.html:32 source_templates/index.html:84
+msgid "Powered by"
+msgstr "مشغل بواسطة"
+
+#: source_templates/base.html:33
+msgid "LOG OUT"
+msgstr "تسجيل الخروج"
+
+#: source_templates/base.html:43 source_templates/index.html:84
+msgid ""
+"Like all software, SecureDrop may contain security bugs. Use at your own "
+"risk."
+msgstr ""
+"ربما يحتوي الSecureDrop على ثغرات أمنية مثل جميع البرمجيات. استخدمها تحت "
+"مسؤوليتك."
+
+#: source_templates/error.html:3
+msgid "Server error"
+msgstr "خطأ في الخادم"
+
+#: source_templates/error.html:5
+msgid ""
+"Sorry, the website encountered an error and was unable to complete your "
+"request."
+msgstr "نأسف, لقد واجه الموقع الإلكتروني خطأ ولم يتمكن من إكمال طلبك."
+
+#: source_templates/error.html:7 source_templates/notfound.html:7
+msgid "Look up a codename..."
+msgstr "ابحث عن اسم رمز..."
+
+#: source_templates/first_submission_flashed_message.html:2
+msgid "Success!"
+msgstr "نجاح!"
+
+#: source_templates/first_submission_flashed_message.html:3
+msgid ""
+"Thank you for sending this information to us. Please check back later for "
+"replies."
+msgstr "شكرا لقيامك بإرسال هذه المعلومات لنا. الرجاء التحقق من الرد لاحقا."
+
+#: source_templates/first_submission_flashed_message.html:5
+msgid "Forgot your codename?"
+msgstr "هل نسيت اسم الرمز الخاص بك؟"
+
+#: source_templates/generate.html:4
+msgid "Welcome"
+msgstr "أهلا بك"
+
+#: source_templates/generate.html:5
+msgid ""
+"This codename is what you will use in future visits to receive messages from "
+"our journalists in response to what you submit on the next screen."
+msgstr ""
+"اسم الرمز هذا ستستخدمه في الزيارات المقبلة لاستلام الرسائل من صحافينا ردا "
+"على ما قمت بتقديمه على الشاشة التالية."
+
+#: source_templates/generate.html:29
+msgid ""
+"Because we use none of the traditional means to track users of our "
+"<strong>SecureDrop</strong>\n"
+" service, using this codename with future visits is the  only way we have to "
+"communicate with you, should we have\n"
+" questions or are interested in additional documents. Unlike passwords, "
+"there is no way to retrieve a lost codename."
+msgstr ""
+"<strong> 1SecureDrop</strong> 2 لأننا لا نستخدم أي من الوسائل التقليدية "
```

Comment:
There are a couple of nits like this `1SecureDrop` in the Arabic translation. I've verified all page layout tests pass, so let's merge in what we have so far and see if we can wrangle up someone to correct any issues in the next few days. Else we will ship what we have.